sound/soc/sof/sof-priv.h

```diff
@@ -388,12 +402,23 @@ struct snd_sof_dai {
 	struct snd_sof_dev *sdev;
 	const char *name;
 	const char *cpu_dai_name;
+	unsigned int pipeline_id;
 
 	struct sof_ipc_comp_dai comp_dai;
 	struct sof_ipc_dai_config *dai_config;
 	struct list_head list;	/* list in sdev dai list */
 };
 
+struct sof_core_ops {
```

Comment:
How are these different to existing ops structures ?

Reply:
@lgirdwood these are functions, that are needed for vhost. In the current model SOF core calls directly functions from the vhost driver. Therefore soc_sof now depends on vhost_sof. That means, that vhost_sof cannot call functions from the SOF core directly. That's why I added these callback ops.

Reply:
then lets make it obvious they are vhost _ops and not generic ops.

Reply:
done